charts/cu-cp/templates/service-e1.yaml

```diff
@@ -1,11 +1,17 @@
+{{- $values := index $.Values "cu-cp" -}}
 {{- include
       "accelleran.common.service"
       (dict
         "top" $
         "values" (mergeOverwrite
-          (deepCopy (index $.Values "cu-cp"))
+          (deepCopy $values)
           (dict "service" (index $.Values "sctp-e1" "service"))
           (dict "service" (dict "name" (include "accelleran.cu-cp.e1.service.name" .)))
```

Comment:
Maybe I m mistaken here but doesn't mergeOverwrite fully replaces the service key here ? I suppose the intention is to add the name to the service?

Reply:
`mergeOverwrite` is the exact same as `merge` except fields at the end get more precedence. So in this case only the name within the service is getting a new value and everything else in the service is kept.

If you would want to fully replace `service` we could for example do something with `omit` to remove everything underneath `service` and then just add the `service.name` again with a `mergeOverwrite` for example.